fpga_interchange/chip_info.py

```diff
@@ -632,6 +632,149 @@ def append_bba(self, bba, label_prefix):
             bba.u32(len(getattr(self, field)))
 
 
+class ChainPattern():
+    def __init__(self, source, sink):
+        self.source = source
+        self.sink = sink
+
+    def append_children_bba(self, bba, label_prefix):
+        pass
+
+    def append_bba(self, bba, label_prefix):
+        bba.str(self.source)
```

Comment:
constids (`str_id`) not raw strings should be being used for stuff like this - have a look at similar places where bel/cell relationships are being described like `LutBel`

Reply:
done, thanks